container.go

```diff
@@ -284,7 +285,7 @@ func (c *container) NewTask(ctx context.Context, ioCreate cio.Creator, opts ...N
 		if err != nil {
 			return nil, err
 		}
-		request.Options = any
+		request.Options = protobuf.FromAny(any)
```

Comment:
request (CreateTaskRequest) is generated from a proto. So we have to call FromAny() here.